config/sync/cfia_base.settings.yml

```diff
@@ -8,8 +8,8 @@ cfia_frontpage:
   frontpage_node_reference: null
 langcode: da
 cfia_footer:
-  footer_text: "<p>Dokk1, Hack Kampmanns Plads 2 8000 Aarhus C -&nbsp;<a href=\"mailto:cfi@aarhus.dk\">cfi@aarhus.dk</a></p>\r\n\r\n<p><a href=\"https://was.digst.dk/cfiaarhus-dk\">Tilgængelighedserklæring</a></p>\r\n\r\n<p>&nbsp;</p>\r\n"
-  footer_twitter: 'https://twitter.com/cfiaarhus'
+  footer_text: "<p><a href=\"https://app.heyloyalty.com/subscribe/10286/3203\" target=\"_blank\">Tilmeld dig vores nyhedsbrev</a>&nbsp;og hold dig opdateret på arrangementer og nyheder fra Center for Innovation Aarhus.&nbsp;| Dokk1, Hack Kampmanns Plads 2 8000 Aarhus C -&nbsp;<a href=\"mailto:cfi@aarhus.dk\">cfi@aarhus.dk</a></p>\r\n\r\n<p><a href=\"https://was.digst.dk/cfiaarhus-dk\">Tilgængelighedserklæring</a></p>\r\n\r\n<p>&nbsp;</p>\r\n"
```

Comment:
You may want to get rid of all that `&nbsp;` stuff in the footer. 

The Web Accessibility Statement should be accessible on `/was` on the site.

Reply:
I made i Jira issue to handle this as a separate issue